src/jockey/geometry.rs

```diff
@@ -0,0 +1,252 @@
+use std::collections::HashMap;
+
+use gl::types::*;
+
+/// A struct represents a single attribute of a Geometry.
+/// You should not use a single GeometryAttribute across multiple Geometries,
```

Comment:
Rust does not allow you to put the same `GeometryAttribute` into multiple `Geometries` like that

Reply:
aha!!